arlo_server/jurisdictions.py

```diff
@@ -7,18 +7,26 @@
     isoformat,
 )
 from arlo_server.models import Jurisdiction
+from util.process_file import serialize_file, serialize_file_processing
 
 
 def serialize_jurisdiction(db_jurisdiction: Jurisdiction) -> dict:
     return {
         "id": db_jurisdiction.id,
         "name": db_jurisdiction.name,
         "ballotManifest": {
-            "filename": db_jurisdiction.manifest_file.name
+            "file": serialize_file(db_jurisdiction.manifest_file)
+            if db_jurisdiction.manifest_file
+            else None,
+            "processing": serialize_file_processing(db_jurisdiction.manifest_file)
             if db_jurisdiction.manifest_file
             else None,
             "numBallots": db_jurisdiction.manifest_num_ballots,
             "numBatches": db_jurisdiction.manifest_num_batches,
+            # Deprecated fields.
+            "filename": db_jurisdiction.manifest_file.name
+            if db_jurisdiction.manifest_file
+            else None,
```

Comment:
I don't think this needs to be backwards compatible. I just added this endpoint and I don't think it's being used yet.

Reply:
Removed.